refs/wg21.json

```diff
@@ -114772,7 +114772,7 @@
         "title": "Minimizing Dependencies within Generic Classes for Faster and Smaller Programs",
         "href": "https://wg21.link/n2911",
         "authors": [
-            "D. Tsafrir, R. Wisniewksi, D. Bacon, B. Stroustrup"
+            "D. Tsafrir, R. Wisniewski, D. Bacon, B. Stroustrup"
```

Comment:
> Dan Tsafrir, Robert W. Wisniewski, David F. Bacon, and Bjarne Stroustrup: Minimizing Dependencies within Generic Classes for Faster and Smaller Programs. ACM OOPSLA'09. October 2009.

https://www.stroustrup.com/papers.html

I'd also claim that it would be best to expand all authors instead of the inconsistent mix of abbreviated/unabbreviated, but I haven't actively done this.

Reply:
This should be split up, too.